packages/engine/src/worker/runner/javascript/apache-arrow-bundle.js

```diff
@@ -1,4 +1,4 @@
-var arrow;
+export var arrow;
```

Comment:
`apache-arrow-bundle` is a vendored in npm-package. Can we use the module directly instead of modifying it?

Reply:
I see, I just tested to remove the `export` but it doesn't work.
I could import it as a script.

Reply:
@leudz We've modified the bundle so that it has the same silly format as our other scripts so it's possible the original bundle might work better. Can you look at the instructions in engine/apache-arrow-js-bundle and regenerate the bundle (making the changes to the classes) and see if importing it works then.

If it doesn't work quickly then lets make a follow-up task and not block this PR on it. Perhaps we need to change the bundle format to ECMA or something

Reply:
I tried to regenerate the bungle and change the bundle options a bit but couldn't make it work.
I made [a task](https://app.asana.com/0/1199548034582004/1202147017388086/f) for it.

Reply:
Do you have documented what you have tried so we don't do the same mistakes again?

Reply:
I didn't...

I remember testing [outputModule](https://webpack.js.org/configuration/experiments/#experimentsoutputmodule) and changing the library type to "umd", "commonjs" and "module".
I wouldn't really exclude what I did from further experiments, I might have missed one thing to configure and didn't get the result we want because of it.

Reply:
Fair enough, thank you.
Marking this as resolved in favor of the [follow-up task](https://app.asana.com/0/1199548034582004/1202147017388086/f) (_internal_).